build/push

```diff
@@ -22,6 +22,30 @@ then
   exit 1
 fi
 
+function pushBuildx() {
+  BUILD_TAG="latest"
+  TARGET_IMG=${DIMAGE}
+
+# TODO Currently ci builds with commit tag will not be generated,
+# since buildx does not support multiple repo
+  # if not a release build set the tag and ci image
+  if [ -z "${RELEASE_TAG}" ]; then
+    return
+#    BUILD_ID=$(git describe --tags --always)
+#    BUILD_TAG="${BRANCH}-${BUILD_ID}"
+#    TARGET_IMG="${DIMAGE}-ci"
+  fi
+
+  echo "Tagging and pushing ${DIMAGE}:${TAG} as ${TARGET_IMG}:${BUILD_TAG}"
```

Comment:
Should TAG be RELEASE_TAG here?

Reply:
Never mind! TAG is set both for release and push builds. RELEASE TAG only for tagged builds. Makes sense.